boards/arm/rak4631_nrf52840/rak4631_nrf52840_defconfig

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_RAK4631_NRF52840=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
```

Comment:
Is `CONFIG_GPIO_AS_PINRESET=y` intentionally ommited here? P0.18 does not seem to be used as a regular GPIO and the module has an NRF_RESET input (so it looks like the pin should be configured as nRESET).

Reply:
you're right